src/java.net.http/share/classes/jdk/internal/net/http/Http2ClientImpl.java

```diff
@@ -293,6 +314,14 @@ SettingsFrame getClientSettings() {
         return frame;
     }
 
+    private void lock() {
+        lock.lock();
+    }
+
+    private void unlock() {
+        lock.unlock();
+    }
```

Comment:
Any specific reason for creating private wrapper methods instead of just using `lock.lock()` and `lock.unlock()` at the call sites?

Reply:
removed private methods